src/python/pants/backend/python/goals/BUILD

```diff
@@ -13,7 +13,9 @@ python_tests(
 
 python_tests(name="coverage_py_test", sources=["coverage_py_test.py"], timeout=20)
 
-python_tests(name="export_integration_test", sources=["export_integration_test.py"])
+python_tests(name="export_test", sources=["export_test.py"])
+
+python_tests(name="export_integration_test", sources=["export_integration_test.py"], timeout=300)
```

Comment:
Running locally this test takes ~120-140 sec. Gave it some breathing room to avoid any flakiness, but I could crank it down